src/Microsoft.VisualStudio.ProjectSystem.Managed.VS.Client/Menus.vsct

```diff
@@ -69,6 +73,11 @@
     <GuidSymbol name="guidWorkspaceExplorerToolWindowPackageCmdSet" value="{cfb400f1-5c60-4f3c-856e-180d28def0b7}">
       <IDSymbol name="idmWSE_ContextMenu" value="0x0002"/>
     </GuidSymbol>
+
+    <!-- Guid and IDs copied from Visual Studio Shell -->
+    <GuidSymbol name="guidManagedProjectSystemClientMainMenuCmdSet" value="{d309f791-903f-11d0-9efc-00a0c911004f}">
```

Comment:
guidSHLMainMenu would be a better description of this. This GUID has nothing to do with us.